src/control/build/version.go

```diff
@@ -92,10 +92,14 @@ func (v Version) Equals(other Version) bool {
 func (v Version) GreaterThan(other Version) bool {
 	if v.Major > other.Major {
 		return true
+	} else if v.Major < other.Major {
+		return false
 	}
 
 	if v.Minor > other.Minor {
 		return true
+	} else if v.Minor < other.Major {
```

Comment:
```suggestion
	} else if v.Minor < other.Minor {
```

Reply:
Sigh. Sorry for asking for review on such sloppy work. I'll fix this and go through it all again to make sure there aren't any other brown paper bag bugs.